result/bgpv4.yaml

```diff
@@ -169,4 +173,24 @@ components:
             openconfirm:
               x-field-uid: 5
             established:
-              x-field-uid: 6
\ No newline at end of file
+              x-field-uid: 6
+        gr_restart_attempted:
+          description: |-
+            Number of Graceful Restarts attempted by the neighbor of the BGP peer. This is incremented
+            when a Open Message received indicating that neighbor has restarted.
+          type: integer
+          format: uint32
+          x-field-uid: 17
+        gr_restart_failed:
+          description: |-
+            Number of Graceful Restarts that were attempted by BGP Peer, however the BGP session
+            was not re-established within the Graceful Restart Time.
+          type: integer
+          format: uint32
+          x-field-uid: 18
+        gr_restart_routes_received:
```

Comment:
I think we wanted to remove this for now till we have more clarity on when this is useful as a metric.